src/main/java/org/isf/opd/model/Opd.java

```diff
@@ -82,10 +82,10 @@
 
 	@NotNull
 	@Column(name="OPD_DATE_VIS")
-	private GregorianCalendar visitDate;
+	private Date visitDate;
```

Comment:
this is correct, `visitDate` is mapped as DATE in the DB, so `java.sql.Date` is the correct mapping and I think the field has been introduced to isolate the date of the visit and then query it without be bothered by the time part.